src/Compilers/CSharp/Portable/Lowering/DiagnosticsPass_Warnings.cs

```diff
@@ -212,7 +212,10 @@ private static bool IsSameLocalOrField(BoundExpression expr1, BoundExpression ex
 
         private static bool IsComCallWithRefOmitted(MethodSymbol method, ImmutableArray<BoundExpression> arguments, ImmutableArray<RefKind> argumentRefKindsOpt)
         {
-            if ((object)method.ContainingType == null || !method.ContainingType.IsComImport) return false;
+            if (method.ParameterCount != arguments.Length ||
+                (object)method.ContainingType == null || 
```

Comment:
This pass happens prior to lowering and can get malformed trees. Lowering will not happen if there are errors, so it would be ok for lowering to make stronger assumptions like every method has containing type.
We, basically, could check if the tree contains errors here and bail, but it is pretty clear from this walker that it intentionally wants to be able to at least try do some work even if we already have errors.